### PR TITLE
feat: add textarea component

### PR DESCRIPTION
**Pull request by mkolasinski-splunk**

### CODE REVIEW

File: pytest_splunk_addon_ui_smartx/components/controls/textarea.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            :return: Str Value of the placeholder
+        """
+        return self.input.get_attribute("placeholder")
+
+    def get_textarea_height(self) -> int:
+        """
+        Get the height of the displayed textarea.
+            :return: Int Height of the textarea in pixels.
+        """
+        style = self.input.get_attribute("style")
```
Comment (vtsvetkov-splunk) [inline pytest_splunk_addon_ui_smartx/components/controls/textarea.py:64]: why not `self.input.size['height']`?
https://github.com/SeleniumHQ/selenium/blob/6359a597c23a43e51142d413f9b23e98c3245d43/py/selenium/webdriver/remote/webelement.py#L272

Comment (mkolasinski-splunk) [reply pytest_splunk_addon_ui_smartx/components/controls/textarea.py]: Used proposed solution

**Comment by mkolasinski-splunk** [comment]
> Can you please update license in the new file? compliance-copyrights is complaining.
> 
> Can you please also add a UI test covering text box in this repo?

License updated, ui tests added.

**Pull request merged**